aten/src/ATen/core/Dict.h

```diff
@@ -103,6 +103,13 @@ class DictEntryRef final {
 template<class Key, class Value, class Iterator>
 class DictIterator final : public std::iterator<std::forward_iterator_tag, DictEntryRef<Key, Value, Iterator>> {
```

Comment:
did you leave the inheritance intentionally?

Reply:
By mistake, thank you for the thorough review